sinonjs/referee-sinon
# Upgrade eslint-config-sinon to latest

This PR upgrades `eslint-config-sinon` to latest and replaces #84 

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm run lint`
1. Observe that there are no errors or warnings reported

#### Checklist for author
